sdk/python/tests/integration/feature_repos/test_repo_configuration.py

```diff
@@ -208,9 +214,14 @@ def construct_test_environment(
     The user is *not* expected to perform any clean up actions.
 
     :param test_repo_config: configuration
+    :param create_and_apply: whether to create and apply the repo config
+    :param materialize: whether to materialize features to online store
+    :param entity_type: the data type for the entity column (i.e. id)
+    :param feature_dtype: the data type for the feature column (i.e. value)
+    :param feature_is_list: whether the feature column (i.e. value) should be a list feature
     :return: A feature store built using the supplied configuration.
     """
-    df = create_dataset()
+    df = create_dataset(entity_type, feature_dtype, feature_is_list)
```

Comment:
I think this is cool for now, but just a note that I'm refactoring this path a bit so the dataset creation is more pluggable to allow for custom datasets a little more easily.

Reply:
Noted. Would you rather I wait to merge this in until after your refactor then?